route/test/test_route_generator.cpp

```diff
@@ -35,6 +35,10 @@
 #include "route/route_state_worker.hpp"
 #include "route/route_node.hpp"
 
+// These tests has been temporarily disabled to support Continuous Improvement (CI) processes.
+// Related GitHub Issue: <https://github.com/usdot-fhwa-stol/carma-platform/issues/2335>
+
+/**
```

Comment:
I thought I'd fixed the issues with this package in a recent PR: https://github.com/usdot-fhwa-stol/carma-platform/pull/2301

Just curious, did CI show that this was failing again, or was it commented-out after reviewing the old unit tests logs posted to the Jira story? If these were commented out due to the older unit test log posted on Jira, we might be able to uncomment these now.

Reply:
yes, this was the first package I updated and just the list. Uncommented the tests.